config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/YourFirstBow-AAAAAAAAAAAAAAAAAAADXg==.json

```diff
@@ -64,7 +64,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Since vanilla bows are only craftable in the assembling machine and mobs don\u0027t drop them very often you should make a Tinkers bow.\n\nWood would be the best material for now. Craft the bow limbs out of wood and the bow string out of strings. The arrow that has the most durability is bone, so craft the stick out of bone. It is much lighter but less durable than a regular arrow. Use a flint tip to make it repairable with flint.\n\n§4You can repair your bow the same way as all the other Tinkers tools. Arrows can be regained by repairing them or picking them up from the floor."
+      "desc:8": "Since vanilla bows are only craftable in the assembling machine and mobs don\u0027t drop them very often you should make a Tinkers bow.\n\nWood would be the best material for now. Craft the bow limbs out of wood and the bow string out of strings. The arrow that has the most durability is bone, so craft the stick out of bone. It is much lighter but less durable than a regular arrow. Use a flint tip to make it repairable with flint.\n\nYou can repair your bow like all other Tinkers tools - with the material you crafted it. Arrows can be regained by repairing them or picking them up from the floor. Keep in mind that they need to be in your inventory to be pickable, not in your Battle Gear slots."
```

Comment:
pickable again? still not sure what you even mean. based on this one maybe "useable"?

Reply:
just tested, battlegear slots work fine. just remove this

Reply:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426#discussion_r1185994629

Reply:
Ok, reverting since it works in dev.

Reply:
Reworded and reinstated in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/4546e78a6b65e97d98d3779f433904e9f4980311